inst/stan/functions/gaussian_process.stan

```diff
@@ -1,7 +1,7 @@
 /**
  * These functions implemente approximuate Gaussian processes for Stan using
  * Hilbert space methods. The functions are based on the following:
- * - https://avehtari.github.io/casestudies/Motorcycle/motorcycle_gpcourse.html#4_Heteroskedastic_GP_with_Hilbert_basis_functions
```

Comment:
does this work as a link now?

Reply:
maybe take that bit out as well then?

Reply:
you mean the mention of the section?

Reply:
the point of the section was that the link took you directly to it. Removing the hash means it is just floating text that has no clear point?

Reply:
it's still telling you where to look, even if not directly linking to it (because we can't)